tasks/windows/2008r2.task/second-stage.ps1.erb

```diff
@@ -0,0 +1,52 @@
+# -*- powershell -*-
+write-host "Razor second-stage Windows installer script starting"
+
+# @todo danielp 2013-11-06: configuration data -- right now we hard-code /
+# guess these because we don't have a way to set this when creating the
+# installer.  That will, I guess, have to change some time soon.  Look to see
+# something akin to the way that brokers work implemented.
+$server = '<%= `hostname`.strip %>'
+$share  = 'razor'
+$repo   = '<%= repo_url.split('/').last %>'
+$drive_path = "\\${server}\${share}\${repo}"
+$installer = "${drive_path}\setup.exe"
+
+write-host "fetching the unattended.xml file to feed the installer"
+$unattended_src = '<%= file_url('unattended.xml') %>'
+$unattended = "${env:SYSTEMDRIVE}\unattended.xml"
+(new-object System.Net.WebClient).DownloadFile($unattended_src, $unattended)
+
+write-host "fetching fastprox.dll from repo to overwrite the local copy (workaround)"
+# This is a workaround for an installer issue with WinPE 4.0 + 2008R2.
+$fastprox_loc = '<%= repo_url('sources/fastprox.dll') %>'
+$fastprox = "${env:SYSTEMDRIVE}\Windows\System32\wbem\fastprox.dll"
+$fastprox_backup = $fastprox + ".bak"
+move $fastprox $fastprox_backup
```

Comment:
It should come bundled in the WinPE environment. There are two error cases:
- The WinPE (original) file might be absent. If this is the case, red text will appear but Powershell will continue executing. I think this is OK behavior.
- The fastprox.dll file on the server may be absent (not provided in ISO (?) or deleted by the user). If this occurs, I'm not sure we have a good fall-back for behavior. I'd expect the install to fail down the line with an error mentioning the missing file.